src/adapter/src/coord/sequencer/inner/create_continual_task.rs

```diff
@@ -100,7 +93,29 @@ impl Coordinator {
             },
         };
 
+        // First thing, assign the id and then rewrite `create_sql` and `expr`
+        // to use it.
+        let full_name = bootstrap_catalog.resolve_full_name(&name, None);
```

Comment:
You should be able to provided `Some(session.conn_id())` here. Might require a bit more elbow grease but this is needed to allow creating `TEMPORARY` continual tasks that get cleaned up at session end

Reply:
Done